packages/node-bridge/build.js

```diff
@@ -4,56 +4,51 @@ const execa = require('execa');
 const { join } = require('path');
 
 async function main() {
+  await fs.remove(join(__dirname, 'helpers.js'));
+  await fs.remove(join(__dirname, 'web-handler.js'));
+
   // Build TypeScript files
   await execa('tsc', [], {
     stdio: 'inherit',
   });
 
   // Bundle `helpers.ts` with ncc
-  await fs.remove(join(__dirname, 'helpers.js'));
-  const helpersDir = join(__dirname, 'helpers');
-  await execa(
-    'ncc',
-    [
-      'build',
-      join(__dirname, 'helpers.ts'),
-      '-e',
-      '@vercel/node-bridge',
-      '-e',
-      '@vercel/build-utils',
-      '-e',
-      'typescript',
-      '-o',
-      helpersDir,
-    ],
-    { stdio: 'inherit' }
-  );
-  await fs.rename(join(helpersDir, 'index.js'), join(__dirname, 'helpers.js'));
-  await fs.remove(helpersDir);
+  await bundle(join(__dirname, 'helpers.ts'), 'helpers');
 
   // Bundle `source-map-support/register` with ncc for source maps
-  const sourceMapSupportDir = join(__dirname, 'source-map-support');
+  await bundle(
+    join(__dirname, '../../node_modules/source-map-support/register'),
+    'source-map-support'
+  );
+
+  // Bundle `web-handler.ts` with ncc
+  await bundle(join(__dirname, 'web-handler.ts'), 'web-handler');
+}
+
+async function bundle(sourceFile, finalName) {
+  const outputDir = join(__dirname, finalName);
   await execa(
     'ncc',
     [
       'build',
-      join(__dirname, '../../node_modules/source-map-support/register'),
+      sourceFile,
       '-e',
       '@vercel/node-bridge',
       '-e',
       '@vercel/build-utils',
       '-e',
       'typescript',
       '-o',
-      sourceMapSupportDir,
+      outputDir,
     ],
     { stdio: 'inherit' }
   );
   await fs.rename(
-    join(sourceMapSupportDir, 'index.js'),
-    join(__dirname, 'source-map-support.js')
+    join(outputDir, 'index.js'),
+    join(__dirname, `${finalName}.js`)
   );
-  await fs.remove(sourceMapSupportDir);
+  console.error(`ncc: Writing ${finalName}.js`);
```

Comment:
*question* is this intentional or leftover debug?

Reply:
It is intentional. It's using stderr because ncc does so, to make it the build output easier to figure out.
With it:
```sh
﻿❯ pnpm --filter node-bridge build

> @vercel/node-bridge@3.1.10 build /Users/damien/dev/vercel/packages/node-bridge
> node build.js

ncc: Version 0.24.0
ncc: Compiling file index.js
ncc: Using typescript@4.3.4 (local user-provided)
 1kB  helpers/helpers.d.ts
 2kB  helpers/types.d.ts
22kB  helpers/index.js
25kB  [1330ms] - ncc 0.24.0
ncc: Writing helpers.js
ncc: Version 0.24.0
ncc: Compiling file index.js
122kB  source-map-support/index.js
122kB  [244ms] - ncc 0.24.0
ncc: Writing source-map-support.js
ncc: Version 0.24.0
ncc: Compiling file index.js
ncc: Using typescript@4.3.4 (local user-provided)
 0kB  web-handler/web-handler.d.ts
 1kB  web-handler/helpers.d.ts
 2kB  web-handler/types.d.ts
10kB  web-handler/index.js
13kB  [1246ms] - ncc 0.24.0
ncc: Writing web-handler.js
```

Without:
```sh
❯ pnpm --filter node-bridge build

> @vercel/node-bridge@3.1.10 build /Users/damien/dev/vercel/packages/node-bridge
> node build.js

ncc: Version 0.24.0
ncc: Compiling file index.js
ncc: Using typescript@4.3.4 (local user-provided)
 1kB  helpers/helpers.d.ts
 2kB  helpers/types.d.ts
22kB  helpers/index.js
25kB  [1157ms] - ncc 0.24.0
ncc: Version 0.24.0
ncc: Compiling file index.js
122kB  source-map-support/index.js
122kB  [236ms] - ncc 0.24.0
ncc: Version 0.24.0
ncc: Compiling file index.js
ncc: Using typescript@4.3.4 (local user-provided)
 0kB  web-handler/web-handler.d.ts
 1kB  web-handler/helpers.d.ts
 2kB  web-handler/types.d.ts
10kB  web-handler/index.js
13kB  [1252ms] - ncc 0.24.0
```

I don't have strong opinion on this, I can rollback.

Reply:
leave it, it's useful information :)